### ISSUE TITLE
Multiple params and object properties

### ISSUE THREAD

**Issue by Kagami** [jsdoc3/jsdoc#917]
Hi! Haven't found similar issue in the search.

---

For this code:
```js
/**
 * Test function.
 * @param {Object} opts - Options
 * @param {number} opts.num2 - Second number
 * @param {number} num - First number
 */
exports.testfun = function() {};
```

`jsdoc` produces correct definition:
![2](https://cloud.githubusercontent.com/assets/533383/6128548/dbe9ddb4-b147-11e4-8edb-b171c7b71720.png)

---

But if options object is last parameter:
```js
/**
 * Test function.
 * @param {number} num - First number
 * @param {Object} opts - Options
 * @param {number} opts.num2 - Second number
 */
exports.testfun = function() {};
```

`jsdoc` doesn't produce nice options table:
![1](https://cloud.githubusercontent.com/assets/533383/6128567/1c8147fe-b148-11e4-9d61-928568ac88c5.png)

**Comment by hegemonic** [comment]
I think this issue has the same root cause as #901, which is fixed on GitHub master. I verified that your second example works correctly on master.

The fix was not in 3.3.0-beta1, but it will be included in the final release of 3.3.0.

**Comment by Kagami** [comment]
Oh, thanks, you are right! Sorry for not testing the master.
